core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -418,6 +418,18 @@ public boolean isMulticastCapable() {
       return channel.getProtocolStack().getTransport().supportsMulticasting();
    }
 
+   @Override
+   public void checkCrossSiteAvailable() throws CacheConfigurationException {
+      if (findRelay2() == null) {
+         throw CLUSTER.crossSiteUnavailable();
+      }
+   }
+
+   @Override
+   public String localSiteName() {
+      return findRelay2().site();
```

Comment:
can throw nullpointer if called without the check

Reply:
nice catch!